docs/user/04-10-footprint.md

```diff
@@ -0,0 +1,16 @@
+# Keda module footprint
+
+The Keda module consists of four workloads.
+Two of them, `keda-manager` and `keda-operator`, implement the Kubernetes operator pattern and consume extra resources only when they detect changes in the resources they watch (for example, Keda CR, ScaledObject CR, etc.). Usually, they are idle and consume as little as few millicores of CPU time and less than 30MB of memory. At the time of active reconciliation, the observed CPU time jumps to 5m.
+
+Similarly to the operators, `keda-admission-webhooks` stays idle most of the time and performs validation operations only when you submit a new object (from *.keda.sh API group).
```

Comment:
```suggestion
Similarly to the operators, `keda-admission-webhooks` stays idle most of the time and performs validation operations only when you submit a new object from `*.keda.sh` API group.
```
Why should this be in paratheses?

Reply:
It's the name of the workload, like `keda-manager`, and `keda-operator`